src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+# Specify the DNS URI of your Azure Cosmos DB.
+azure.cosmosdb.uri=https://localhost:8081
+# Specify the access key for your database.
+azure.cosmosdb.key=C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==
```

Comment:
@overlxrd , is there some script you used to startup the local database and set its password? If so, it would be worth adding that into this PR as well.

Reply:
The database config has since been changed. Details on running locally will be in the README